rclpy/src/rclpy/_rclpy.c

```diff
@@ -634,6 +635,56 @@ rclpy_init(PyObject * Py_UNUSED(self), PyObject * args)
   Py_RETURN_NONE;
 }
 
+/// Initialize rcl logging
+/**
+ * Raises RuntimeError if rcl logging could not be initialized
+ */
+static PyObject *
+rclpy_logging_configure(PyObject * Py_UNUSED(self), PyObject * args)
+{
+  // Expect one argument, a context.
+  PyObject * pycontext;
+  if (!PyArg_ParseTuple(args, "O", &pycontext)) {
+    // Exception raised
+    return NULL;
+  }
+  rcl_context_t * context = rclpy_handle_get_pointer_from_capsule(pycontext, "rcl_context_t");
+  if (!context) {
+    return NULL;
+  }
+  rcl_allocator_t allocator = rcl_get_default_allocator();
+  rcl_ret_t ret = rcl_logging_configure(
+    &context->global_arguments,
+    &allocator);
```

Comment:
@ivanpauno nit: do we need to wrap this function call's arguments?

Reply:
No, but I think both styles are accepted in our guidelines.
I fixed it in https://github.com/ros2/rclpy/pull/518/commits/a5814ed5b7c5788ecd6e77aa11fa36580ed8efaa.